bg_mon.c

```diff
@@ -358,6 +359,16 @@ static struct evbuffer *prepare_statistics_output(struct timeval time, system_st
 	evbuffer_add_printf(evb, "{\"hostname\":\"%s\",\"time\":%llu,\"sysname\":\"Linux: %s\",", s.hostname, ts, s.sysname);
 	evbuffer_add_printf(evb, "\"cpu_cores\":%d,\"postgresql\":{\"version\":\"%s\"", c.cpu_count, PG_VERSION);
 	evbuffer_add_printf(evb, ",\"role\":\"%s\",", p.recovery_in_progress?"replica":"master");
+
+	if (p.recovery_in_progress) {
+		evbuffer_add_printf(evb, "\"wal\":{\"replay_paused\":%s,", w.is_wal_replay_paused?"true":"false");
+		evbuffer_add_printf(evb, "\"replay_timestamp\":%ld,", timestamptz_to_time_t(w.last_xact_replay_timestamp));  /* unix seconds! */
+		evbuffer_add_printf(evb, "\"receive\":%lu,", w.receive_diff);
+		evbuffer_add_printf(evb, "\"replay\":%lu},", w.replay_diff);
+	} else {
+		evbuffer_add_printf(evb, "\"wal\":{\"written\":%lu},", w.current_diff);
```

Comment:
Same as for https://github.com/CyberDem0n/bg_mon/pull/39/files#diff-57f1b76e196394f418a39a1f6d499e8da40bc02a013bf7db13a092fa79c1a60aR364